.github/workflows/tests.yaml

```diff
@@ -37,11 +40,11 @@ jobs:
           snap watch --last=install
           echo "::endgroup::"
       - name: Run Linters
-        run: tox run -m lint
+        run: .tox/.tox/bin/tox run -m lint
```

Comment:
Why did you need the path?

Reply:
This uses the copy of tox inside the environment provisioned based on tox.ini so we don't get the logging noise of telling us that it's provisioning the environment. Combined with `--no-list-dependencies` in [the next tox release](https://github.com/tox-dev/tox/pull/3024) this will keep the CI noise in running tox to a minimum.